### ISSUE TITLE
[feature request] CLI --quiet switch

### ISSUE THREAD

**Issue by FredLackeyOfficial** [adamreisnz/replace-in-file#63]
Please add a `--quiet` switch for the CLI wrapper.  I would prefer to use this in a script without any output upon success.

**Comment by adamreisnz** [comment]
I don't think we log anything from the script itself, only if you use `dry` mode: https://github.com/adamreisnz/replace-in-file/blob/master/lib/replace-in-file.js#L37-L39

Can you show me what output you get?

**Comment by FredLackeyOfficial** [comment]
```
$ node_modules/.bin/replace-in-file "/\{NdefaultValue.*?$/" "" data/models/**/*.js
Replacing '/\{NdefaultValue.*?$/' with ''
No files were changed
```

**Comment by adamreisnz** [comment]
Implemented in 4.0.0
